course/templates/course/_siblings.html

```diff
@@ -10,14 +10,14 @@
         {% module_accessible next as accessible %}
     {% endif %}
     {% if accessible %}
-    <a href="{{ next.link }}" class="pull-right btn btn-link">
+    <a href="{{ next.link }}" class="pull-right">
         {{ next.name|parse_localization }} &raquo;
     </a>
     {% else %}
-    <span class="pull-right btn btn-link disabled">
+    <span class="pull-right disabled">
         {{ next.name|parse_localization }} &raquo;
         {% if is_course_staff %}
-        <a href="{{ next.link }}" class="btn btn-default btn-xs">
+        <a href="{{ next.link }}" class="aplus-button--secondary aplus-button--xs">
```

Comment:
That's a good point. 

Fixed now (although the link text is a little cryptic, in my opinion).

Reply:
The "early access" links are someone's historical decision. I think it would be better to just render the link in the exercise title to teachers without a separate "teacher's early access link". But we won't change that in this PR.